roles/container-engine/nerdctl/tasks/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: install nerdctl
+  include_tasks: nerdctl.yml
+  when: nerdctl_enabled
```

Comment:
Why this weird include_tasks, please put everything in main.yml

Reply:
removed nerdctl.yml and put it in mani.yaml.